File: taskmanager/src/test/java/com/example/alexeyglushkov/taskmanager/task/StackTaskProviderWithRestorableProviderTest.java
package com.example.alexeyglushkov.taskmanager.task;

import android.os.Handler;
import android.os.Looper;

/**
 * Created by alexeyglushkov on 18.06.17.
 */

public class StackTaskProviderWithRestorableProviderTest extends StackTaskProviderTest{
    @Override
    protected TaskProvider prepareTaskProvider() {
        return new RestorableTaskProvider(new StackTaskProvider(false, new Handler(Looper.myLooper()), "TestId"));
    }
}

File: taskmanager/src/test/java/com/example/alexeyglushkov/taskmanager/task/StackTaskProviderWithRestorableProviderTest.kt
package com.example.alexeyglushkov.taskmanager.task

import android.os.Handler
import android.os.Looper

/**
 * Created by alexeyglushkov on 18.06.17.
 */

class StackTaskProviderWithRestorableProviderTest : StackTaskProviderTest() {
    override fun prepareTaskProvider(): TaskProvider {
        return RestorableTaskProvider(StackTaskProvider(false, Handler(Looper.myLooper()), "TestId"))
    }
}
